tests/kernel/wave/wave_e2e_test.py

```diff
@@ -290,7 +291,7 @@ def test(
         run_config=config,
     ):
         test(a, b, c)
-        assert_allclose(ref.values, c)
+        assert torch.equal(c, ref.values)

```

Comment:
Do we expect tensors to be bit-exact? Why not assert_equal?

Reply:
for max, yes :)

Reply:
Oh, wait, I didn't noticed it was max ), anyway, why not `assert_equal`?

Reply:
Keeping it uniform with the wave_e2e_gemm test

Reply:
`assert_equal` is better, it will actually print array contents and number of elements mismatched.

Reply:
Done!